.development-configs/checkstyle.xml

```diff
@@ -108,6 +100,9 @@
       <property name="ignoreNumbers" value="-1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 100, 128, 256, 512, 768, 1024, 2048, 4096"/>
     </module>
   </module>
+  <module name="LineLength">
+    <property name="max" value="120"/>
```

Comment:
120 is certainly more sensible than 80; it's the value e.g. the linux kernel codestyle also advises :+1:

Reply:
80 is good in C, but Java tends to have longer names.
I prefer 100 as good breaking point for Java, but sometimes it's just cleaner to go with 120, so I think it's best to enforce 120.